hibernate-core/src/main/resources/org/hibernate/hibernate-mapping-3.0.dtd

```diff
@@ -135,6 +135,7 @@ arbitrary number of queries, and import declarations of arbitrary classes.
 <!ELEMENT filter-def (#PCDATA|filter-param)*>
 	<!ATTLIST filter-def name CDATA #REQUIRED> <!-- The filter name -->
 	<!ATTLIST filter-def condition CDATA #IMPLIED>
+	<!ATTLIST filter-def auto-enabled (true|false) "false">
```

Comment:
Please revert the changes to this file. hbm.xml is deprecated and will most likely be removed in 7.0.